### PR TITLE
[FEATURE]: Elevate Root

### PR DESCRIPTION
**Pull request by amadolid**
Support elevate current root to be system root

### CODE REVIEW

**Comment by ChrisIsKing** [comment]
@amadolid Let's say you wanted to elevate to root access by from the python side instead of jac, what would be the interface to do that? Like if I wanted to run `_Jac.spawn_call()` programmatically.

**Comment by amadolid** [comment]
> @amadolid Let's say you wanted to elevate to root access by from the python side instead of jac, what would be the interface to do that? Like if I wanted to run `_Jac.spawn_call()` programmatically.

it should be the same `Jac.elevate_root()` or `_Jac.elevate_root()`

File: jac/jaclang/plugin/default.py [hunk 1]
```diff
@@ -54,6 +54,13 @@
 class JacAccessValidationImpl:
     """Jac Access Validation Implementations."""
 
+    @staticmethod
+    @hookimpl
+    def elavate_root() -> None:
```
Comment (ChrisIsKing) [inline jac/jaclang/plugin/default.py]: Everything looks fine, just that it should be spelt `elevate_root`

**Review by ChrisIsKing — CHANGES_REQUESTED** [review]

File: jac/jaclang/plugin/default.py (hunk 1 above)
Comment (ypkang) [reply jac/jaclang/plugin/default.py]: @amadolid ping on this.

**Comment by amadolid** [comment]
Rename and rebased

**Review by ypkang — APPROVED** [review]

**Review by ChrisIsKing — APPROVED** [review]

**Pull request merged**